docs/development-guide.md

```diff
@@ -24,6 +24,8 @@ this relative directory layout.
 - docker
 - make
 - Python 3 (only for needed for deploying)
+- npm
+- docker-compose, with a more modern version than Debian's defaults (https://docs.docker.com/compose/install/)
```

Comment:
yes, good point, `docker-compose` has to be installed separately from `docker` and really should be in this list. I don't think I'd mention Debian here, since this should be OS agnostic.
OS specific sections may be warranted at some point.

Reply:
Good point. I was unable to determine the maximum version of docker's YML that we're using, only that the one in Mint/Ubuntu/Debian's repo is very far out of date. If you know the maximum YML version we're using, I can translate that into a minimum docker-compose version.

Reply:
Actually I think I've set the version in the docker-compose.yml files (`version: '3.6'` is what i see that I've specified there)

Reply:
Perfect. I'll add the version 18.02.0+ for docker-compose.